__tests__/app.test.js

```diff
@@ -40,6 +40,56 @@ describe("GET", ()=>{
                 })
         })
     })
+    describe("/api/reviews", () => {
+        it("should return status: 200, and return a list of all reviews with specific properties", () => {
+            return request(app)
+                .get("/api/reviews")
+                .expect(200)
+                .then(({body}) => {
+                    body.reviews.forEach((review) => {
```

Comment:
we need to do a length check of the array to make sure it's not empty (that would give a false positive)